components/widgets/users/avatar/avatar.scss

```diff
@@ -0,0 +1,74 @@
+.Avatar {
+    border-radius: 50%;
+    -webkit-user-select: none; /* Chrome all / Safari all */
+    -moz-user-select: none;    /* Firefox all */
+    -ms-user-select: none;     /* IE 10+ */
+    user-select: none;
```

Comment:
Purely as an aside, we really need to get our tooling to handle this! 😛

Reply:
Absolutely! Only showing in the diff here cause I moved the component/files into a folder.

Reply:
And perhaps we don't even need them now that we've dropped IE and all the other browsers should be respecting this (any many other previously prefixed properties) now.

Reply:
Ya, we will need to remove them all at some point ... don't worry about now unless my mentioning it is now bugging you! :P